src/mono/wasm/wasm.proj

```diff
@@ -125,10 +134,14 @@
 
     <ItemGroup>
       <_EmccLinkFlags Include="-s EXPORT_ES6=1" Condition="'$(WasmEnableES6)' == 'true'" />
+      <_EmccLinkFlags Include="-s LLD_REPORT_UNDEFINED" />
```

Comment:
intended?

Reply:
Looking at the Emscripten release notes, we probably don't want this on by default.  @steveisok what was the intention here?  Just validating that all the EP stuff got built/linked?

Reply:
It was used to potentially root out issues.  We can remove it or if it makes sense, enable by flag.